meshnets/utils/datasets.py

```diff
@@ -1,90 +1,197 @@
 """File containing dataset classes."""
 import os
+import warnings
 
 import numpy as np
 import torch
 from torch_geometric.data import Data, Dataset
 
+from meshnets.utils import data_processing
 
-class FromDiskDataset(torch.utils.data.Dataset):
-    """Reads a dataset from a given directory.
+# TODO(victor): the wind vector should be
+# obtained individually for each mesh
+WIND_VECTOR = (10, 0, 0)
```

Comment:
I actually don't think we should use the wind vector as node features. It is adding unnecessary calculations since the wind vector is the same for all nodes. It should either be a global feature of the graph, or at this stage (with one wind velocity vector only) none at all. Let's open an issue to discuss this further.

Reply:
Right, but for now I guess we can see that as a placeholder that we can later change when augmenting the model dynamics.

Reply:
Agreed. Let's just open the issue so we don't forget that the wind vector is being used as a node feature, equal for all nodes.